airflow/models/backfill.py

```diff
@@ -335,15 +353,21 @@ def _create_backfill_dag_run(
         except IntegrityError:
             log.info(
                 "Skipped creating backfill dag run for dag_id=%s backfill_id=%s, logical_date=%s (already exists)",
-                dr.dag_id,
-                dr.id,
+                dag.dag_id,
+                backfill_id,
                 info.logical_date,
             )
-            log.info("Doing session rollback.")
-            session.rollback()
+            log.info("rolling back")
```

Comment:
Do we need 2 logs? Can we just wrap this into the previous info? This one feels a bit sparse.

Reply:
this actually seems more like debug

Reply:
if anything

Reply:
```suggestion
```